Sources/SwiftDocC/Model/Rendering/RenderSectionTranslator/DeclarationsSectionTranslator.swift

```diff
@@ -162,11 +206,12 @@ struct DeclarationsSectionTranslator: RenderSectionTranslator {
                     let processedOverloadDeclarations = overloadDeclarations.map({
                         OverloadDeclaration($0.declaration.flatMap(preProcessFragment(_:)), $0.reference)
                     })
-                    let preProcessedDeclarations = [mainDeclaration] + processedOverloadDeclarations.map(\.declaration)
 
                     // Collect the "common fragments" so we can highlight the ones that are different
                     // in each declaration
-                    let commonFragments = longestCommonSubsequence(preProcessedDeclarations)
+                    let commonFragments = commonFragments(
```

Comment:
Again, it appears you only ever need the common fragments for the main declaration, and not for the individual overloads.